tests/models/dpt/test_image_processing_dpt.py

```diff
@@ -138,3 +168,124 @@ def test_keep_aspect_ratio(self):
         pixel_values = image_processor(image, return_tensors="pt").pixel_values
 
         self.assertEqual(list(pixel_values.shape), [1, 3, 512, 672])
+
+    def test_call_segmentation_maps(self):
```

Comment:
Same as before about the # Copied from, if this and the following tests are indeed fully copied from beit